VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/host-hearing-list/host-hearing-list.component-base.ts

```diff
@@ -15,12 +14,8 @@ import { pageUrls } from 'src/app/shared/page-url.constants';
 import { ScreenHelper } from 'src/app/shared/screen-helper';
 import { HearingRole } from 'src/app/waiting-space/models/hearing-role-model';
 
-@Component({
-    selector: 'app-judge-hearing-list',
-    templateUrl: './judge-hearing-list.component.html',
-    styleUrls: ['./judge-hearing-list.component.scss']
-})
-export class JudgeHearingListComponent implements OnInit, OnDestroy {
+@Directive()
+export abstract class HostHearingListComponent implements OnInit, OnDestroy {
```

Comment:
Maybe change to HostHearingListBaseComponent

Reply:
yea I will rename it

Reply:
renaming this as HostHearingListBaseComponentDirective because our lint requires anything decorated with @Directive() to end with a suffix of Directive. 😑

Reply:
I see. Well technically a component is a type of directive I guess, so that's fine.